### PR TITLE
Add Moonlight Scheme

### PR DESCRIPTION
**Pull request by mauroreisvieira**

### CODE REVIEW

File: repository/m.json [hunk 1]
```diff
@@ -2354,6 +2354,16 @@
 				}
 			]
 		},
+		{
+			"name": "Moonlight",
+			"details": "https://github.com/mauroreisvieira/moonlight-sublime-theme",
+			"releases": [
```
Comment (braver) [inline repository/m.json:2360]: Please add `"labels": ["color scheme"]`

**Comment by braver** [comment]
Please have a look at the results from the bot, and maybe tell us a little bit about your package.

**Comment by mauroreisvieira** [comment]
Hey @braver thanks for the feedback.



I already add the `label` as you suggested, but I still don't understand the issue with key `install` because I have a message folder with `install.md` file and in my `message.json` I have this

```json
{
  "install": "messages/install.md",
  "1.0.0": "messages/1.1.0.md"
}
```

I have the same approach w/out any error in:

- [Meetio Theme](https://github.com/meetio-theme/sublime-meetio-theme/blob/main/messages.json)
- [GitHub Theme](https://github.com/mauroreisvieira/github-sublime-theme/blob/main/messages.json)

can you help me to understand what is my issue? 🤷🏻‍♂️

**Comment by braver** [comment]
Stupid bot 🤷🏼‍♂️

**Pull request merged**